.gitignore

```diff
@@ -9,3 +9,4 @@ target
 *.iws
 .idea
 .DS_Store
+*.monopic
```

Comment:
I'm wondering if this is required in this PR.

Reply:
Removed